hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalReplicatedMapStatsImpl.java

```diff
@@ -309,6 +312,24 @@ public NearCacheStatsImpl getNearCacheStats() {
         throw new UnsupportedOperationException("Replicated map has no Near Cache!");
     }
 
+    @Override
+    public long getQueryCount() {
+        // there is no query support for replicated maps
+        return 0;
```

Comment:
Why don't we throw `UnsupportedOperationException` for these three methods? I think that would prevent users from falsely assuming there is support for it.

Reply:
fixed